WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs

```diff
@@ -45,7 +44,7 @@ public CoinViewModel(CoinListViewModel owner, SmartCoin model)
 
 			Model = model;
 			Owner = owner;
-			InCoinJoinContainer = owner.CoinListContainerType == CoinListContainerType.CoinJoinTabViewModel;
+			CanBeDequeued = owner.CanDequeueCoins;
```

Comment:
Once the search has began something will be found: `public bool CanBeDequeued => Owner.CanDequeueCoins`.

Reply:
done